.travis.yml

```diff
@@ -1,3 +1,5 @@
+sudo: required
+dist: trusty
```

Comment:
Before I added this line the tests still were run on `precise`:
Before: https://travis-ci.org/sk-/git-lint/jobs/264042669
After: https://travis-ci.org/sk-/git-lint/jobs/264043656
I guess they still run legacy jobs with `precise` to keep them compatible (when I started the same test/commit in my private repo with travis it was already run on `trusty`).
Quote from the [blog post](https://blog.travis-ci.com/2017-07-11-trusty-as-default-linux-is-coming): `complete default dist: trusty rollout | Friday, September 1 2017`, so I guess it is required for the moment.